treeherder/model/management/commands/cycle_data.py

```diff
@@ -21,6 +24,11 @@
 TREEHERDER_SUBCOMMAND = 'from:treeherder'
 PERFHERDER_SUBCOMMAND = 'from:perfherder'
 MINIMUM_PERFHERDER_EXPIRE_INTERVAL = 365
+EMAIL_RECEIVER_TEAM = "perftest-alerts@mozilla.com"
+EMAIL_RECEIVER = "dhunt@mozilla.com"
+
+client_id = settings.PERF_SHERIFF_BOT_CLIENT_ID
```

Comment:
You'll need new separate env variables for emailing. So that we respect the principle of least privilege.